chia/plotting/manager.py

```diff
@@ -81,6 +103,32 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         self._lock.release()
 
+    def cache_path(self):
+        return Path(self.root_path / "plot_manager.dat")
```

Comment:
this is an example of why we need testnet and mainnet to use different directories.. (since this filename isn't parameterized by the network)

Reply:
> we need testnet and mainnet to use different directories

Don't we? I have the file in my `chia/mainnet` folder :thinking:

Reply:
yeah, we haven't gotten around to fix that properly yet. :/

Reply:
If this is a pure cache file, can we add `cache` to the file name to more clearly hint to users it can be safely deleted? Maybe `plot_manager_cache.dat` or `plot_manager.cache` or something. Or put it in a `cache` subdirectory under the root, since this may not be the last cache file.